homeassistant/components/deconz/light.py

```diff
@@ -220,11 +220,15 @@ async def async_turn_on(self, **kwargs: Any) -> None:
         elif "IKEA" in self._device.manufacturer:
             data["transition_time"] = 0
 
-        if (alert := FLASH_TO_DECONZ.get(kwargs.get(ATTR_FLASH))) is not None:
+        if (
+            alert := FLASH_TO_DECONZ.get(cast(str, kwargs.get(ATTR_FLASH)))
```

Comment:
We shouldn't cast None to a string. We probably need to rewrite the expression.

Reply:
How about `if (alert := FLASH_TO_DECONZ.get(kwargs.get(ATTR_FLASH, ""))) is not None:`

Reply:
https://github.com/home-assistant/core/pull/68143